### PR TITLE
Milestone #5: Implement styling

### PR DESCRIPTION
**Pull request by lordksix**
In this PR, the changes are as follows:
- Add final styling
- Improve UI
- Improve add book UX
- Add circular progress bar
- Add simple finished chapter generator

### CODE REVIEW

File: src/styles/App.css [hunk 1]
```diff
@@ -4,36 +4,61 @@
...
+:root {
+  --pale-grey: #f5f6fa;
+  --black-5: rgba(0, 0, 0, 0.05);
+  --white: #fafafa;
+  --white-two: #fff;
+  --azure: #0290ff;
+  --white-three: #e8e8e8;
+  --very-light-pink: #c4c4c4;
+  --medium-blue: #307bbe;
+}
```
Comment (CollinsTatang) [inline src/styles/App.css]: - [x] Please your styling should be exactly the same as the given design on [Zeplin](https://app.zeplin.io/)
### **Take a Look at your design**
Screenshot 2023-06-16 at 04 29 03

### **Zeplin Design**
Screenshot 2023-06-16 at 04 25 17

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @lordksix , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/App.css (hunk 1 above)
Comment (lordksix) [reply src/styles/App.css]: Added mixing icon, fix button size and border, delete inner text in progress bar, align progress text with progress bar, fix upper case letters, and delete category selector

**Review by mckent05 — APPROVED** [review]
Hi @lordksix, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job styling your project according to the design template 👍 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**